src/tests/ftest/deployment/critical_integration.py

```diff
@@ -74,22 +74,22 @@ def test_passwdlessssh_versioncheck(self):
                 if check_remote_root_access:
                     run_command(remote_root_access)
                 IorTestBase._execute_command(self, command_for_inter_node, hosts=[host])
-            except (DaosTestError, CommandFailure) as error:
+            except (DaosTestError, CommandFailure, KeyError) as error:
```

Comment:
Along with this, we should change the above
```
                if 'response' in out:
                    if 'version' in out['response']:
                        daos_server_version_list.append(out['response']['version'])
```
to
```
                daos_server_version_list.append(out['response']['version'])
```
Otherwise, we're still silently skipping when the response is bad

Reply:
ah Okay. Done.